.NET/Microsoft.Recognizers.Text.DateTime/Hindi/Extractors/HindiDurationExtractorConfiguration.cs

```diff
@@ -60,12 +60,15 @@ public HindiDurationExtractorConfiguration(IDateTimeOptionsConfiguration config)
             : base(config)
         {
             CardinalExtractor = Number.Hindi.CardinalExtractor.GetInstance();
+            OrdinalExtractor = null;
```

Comment:
Why is this one null?

Reply:
In this way the pattern is not implemented in Hindi (and Turkish). As commented in the PR description, there are issues in these languages given by the fact that "first" can be interpreted both as ordinal and as "before/previous".